### PR TITLE
fix the issue 390

### PR DESCRIPTION
**Pull request by ManasaBM1**

### CODE REVIEW

**Comment by ManasaBM1** [comment]
Reviewers is @IswaryaaS

File: server/service/individualServices/ElasticsearchPreparation.js [hunk 1]
```diff
@@ -54,7 +77,7 @@ async function createIndexTemplate() {
             index_patterns: `${indexAlias}-*`,
             template: {
                 settings: {
-                    'index.lifecycle.rollover_alias': indexAlias
```
Comment (V-Vanitha) [inline server/service/individualServices/ElasticsearchPreparation.js:57]: Thank you for the changes . But I guess the line is deleted by mistake could you please recheck and make changes.

File: server/service/ElasticsearchClientService.js [hunk 2]
```diff
@@ -69,7 +69,7 @@ exports.getElasticsearchClientOperationalState = async function(uuid) {
 exports.getElasticsearchClientServiceRecordsPolicy = async function(uuid) {
   var value = await elasticsearchService.getElasticsearchClientServiceRecordsPolicyAsync(uuid);
   var response = {};
-  response['application/json'] = {
```
Comment (V-Vanitha) [inline server/service/ElasticsearchClientService.js:72]: This change is already part of PR https://github.com/openBackhaul/OamLog/pull/389/files which is fix for issue https://github.com/openBackhaul/OamLog/issues/387 we can remove changes here

**Review by V-Vanitha — CHANGES_REQUESTED** [review]
can you

**Review by V-Vanitha — APPROVED** [review]

**Pull request merged**